Lesson_1/Task1/Task1.3/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+// Сцепление строк. (Конкатенация)
+
+namespace Concatenation
+{
+    class Program
+    {
+        static void Main()
+        {
+
+            string ph1 = "po";
+            string ph2 = "be";
+            string ph3 = "da";
+
+            Console.WriteLine($"String {ph1.ToUpper()} starts from {ph2.ToUpper()}: {ph1.StartsWith("be")}");
```

Comment:
Great interpolation, but you shouldn't "HARDCODE" the result in ph1.StartsWith("be") here, instead of "be" shoul be variable